File: fizzbuzz.py
# FizzBuzz

# Getting input from users for the max number to go to for FizzBuzz
ip = input("Enter the max number to go for FizzBuzz: \n")

# If user inputs a number this is executed
if (ip != ""):
    for i in range(1, int(ip) + 1):
        checker = i
        if (i % 3 == 0 and i%5 == 0):
            print("FizzBuzz")
        elif (i % 3 == 0):
            print("Fizz")
        elif (i % 5 == 0):
            print("Buzz")
        else:
            print(i)     

# If the input is empty, the value is taken as 100
else:
    for i in range(1, 101):
        checker = i
        if (i % 3 == 0 and i%5 == 0):
            print("FizzBuzz")
        elif (i % 3 == 0):
            print("Fizz")
        elif (i % 5 == 0):
            print("Buzz")
        else:
            print(i)    

File: Java/fizzbuzz.java
/**
 * fizzbuzz
 */

public class fizzbuzz {

    public static void main(String[] args) {
        // Used the MakeJavaEasy library --> https://github.com/virejdasani/Make-Java-Easy
        int input = makejavaeasy.getInt("Enter the max number to go for FizzBuzz: ");

        // For loop in range of input
        for (int i = 0; i < input + 1; i++) {
            // Check for FizzBuzz
            if (i % 3 == 0 && i % 5 == 0) {
                System.out.println("FizzBuzz");
            }
            else if (i % 3 == 0) {
                System.out.println("Fizz");
            }
            else if(i % 5 == 0) {
                System.out.println("Buzz");
            }
            else {
                System.out.println(i);
            }
        }
        
    }
}
